relay-general/src/store/normalize/contexts.rs

```diff
@@ -112,10 +115,20 @@ fn normalize_os_context(os: &mut OsContext) {
                 .name("build")
                 .map(|m| m.as_str().to_string().into())
                 .into();
+        } else if let Some(captures) = OS_LINUX_DISTRO_UNAME_REGEX.captures(raw_description) {
```

Comment:
nit: Do we have any stats on what OS is the most frequent? If Linux is much more frequent than MacOS, should we attempt to parse it first?